### DISCUSSION TITLE
Enable Vue devtools in directus/directus

### DISCUSSION THREAD

**Discussion by zghib** [directus/directus#2728] [Archive]
Hi everybody, here I have a question about enabling Vue devtools. 
So what I'm trying 
1. clone directus/directus
2. run php -S 127.0.0.1:8888 in directus/public
3. deploy a custom module with  'npm run dev'
4. After this, when I'm browsing my custom module I'm expecting to have Vue devtools enabled
But it shows me 
'Vue.js is detected on this page.
Devtools inspection is not available because it's in production mode or explicitly disabled by the author.'

Do you have instruction about how to enable Vue devtools?
I suppose that I need to rebuild the whole /admin with --dev but not sure. 
So, if you'll provide a short how-to it will be very handy.

**Comment by rijkvanzanten** [comment]
You could run the app in dev mode locally for now. I'm also down to accept a PR that enables the Vue devtools in production builds at all times, I don't think  🙂
